scanapi/templates/markdown.jinja

```diff
@@ -2,38 +2,56 @@
 
 {% for response in responses %}
 {% set request = response.request %}
-## Request: {{ request.method }} {{ request.url }}
+## {{ request.method }} {{ request.url }}
```

Comment:
It would be better to not display the full URL but just the path instead. Good for the 👀.
Something we can do is to pass `BASE_URL` to the template and strip it from `request.url`.

Reply:
I strongly agree! We can show the base url only once in the top of the file.